example/readFromString/readFromString.cpp

```diff
@@ -0,0 +1,33 @@
+#include "json/json.h"
+#include <iostream>
+/*
+parse a string to Value object with CharReaderBuilder class or Reader class
+$g++ readFromString.cpp -ljsoncpp -std=c++11 -o readFromString
+$./readFromString
+colin
+20
+*/
+int main() {
+  std::string strRes = R"({"Age": 20, "Name": "colin"})";
```

Comment:
This should be const, and a better name, e.g.

`const std::string rawJson = R"({"Age": 20, "Name": "colin"})";`